python/example/single_cell_allen.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
```

Comment:
This is a good example, but not a good tutorial.

Maybe we have to go back to the structure of a `tutorial` path, which has the tutorial code that leads the user through the process from a single cell to a full model.

Once a user has learnt the interface from the tutorial, or by reading the concepts docs, they can read an example like this that shows the messy process of loading inputs from something like Allen SDK.

Reply:
See above.